buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -391,7 +391,7 @@ class BeamModulePlugin implements Plugin<Project> {
 
     // Automatically use the official release version if we are performing a release
     // otherwise append '-SNAPSHOT'
-    project.version = '2.38.0.8'
+    project.version = '2.38.0.8-test'
```

Comment:
use this [script](https://github.com/linkedin/beam/blob/li_trunk/update_version.sh) to update the version to one only containing numbers like `200.38.0.8`. LI auto dependency import system does not like non-numbered versions

Reply:
Fixed it.